docs/Using-Fleet/CIS-Benchmarks.md

```diff
@@ -0,0 +1,77 @@
+<img src=https://user-images.githubusercontent.com/2073526/220428249-7a1b6433-24fe-4686-8dfb-b555c199f47d.png />
+
+All CIS Benchmarks implemented by Fleet are limited to a Fleet Premium or Fleet Ultimate license.
+
+The Center for Internet Security website offers documentation for all CIS Benchmarks in PDF format. Such PDFs document all the checks, their description, rationale and how to remediate them.
+
+Fleet has implemented CIS benchmarks for the following platforms:
+- CIS Apple macOS 13.0 Ventura Benchmark v1.0.0 - 11-14-2022 (82 checks) 
+- CIS Microsoft Windows 10 Enterprise Benchmark v1.12.0 - 02-15-2022 (Coming Soon!)
+
+## Manual vs Automated
+
+There are two types of CIS Benchmark checks, "Manual" and "Automated".
+- Automated: Represents recommendations for which assessment of a technical control can be fully automated and validated to a pass/fail state
+- Manual: Represents recommendations for which assessment of a technical control cannot be fully automated and requires all or some manual steps to validate that the configured state is set as expected.
+
+Fleet only implements "Automated" checks. "Manual" checks cannot be automated and/or require administrators to decide what's best for their organization's requirements.
+
+## Example CIS check
+
+Sample: `CIS - Ensure Install of macOS Updates Is Enabled` for macOS 13 Ventura Benchmark v1.0.0.
+
+<img src=https://user-images.githubusercontent.com/2073526/220428662-b324f6b7-7209-49a5-ae9f-4131316d8d1a.png />
+
+Such policy corresponds with the following entry in [cis-policy-queries.yml](https://raw.githubusercontent.com/fleetdm/fleet/main/ee/cis/macos-13/cis-policy-queries.yml):
+```yml
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure Install of macOS Updates Is Enabled (MDM Required)
+  platforms: macOS
+  platform: darwin
+  description: Ensure that macOS updates are installed after they are available from Apple.
+  resolution: "Ask your system administrator to deploy an MDM profile that enables automatic install of macOS updates."
+  query: SELECT 1 FROM managed_policies WHERE domain='com.apple.SoftwareUpdate' AND name='AutomaticallyInstallMacOSUpdates' AND value=1 LIMIT 1;
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS-macos-13-1.4
+  contributors: sharon-fdm
+```
+
+- The `description` briefly explains the check and its rationale.
+- The `resolution` explains how administrators can remediate a failing check on a host.
+- The `query` field is the osquery query that will execute on devices.
+
+## Requirements
+
+Following are the requirements to use the CIS Benchmarks in Fleet:
+
+- Fleet must be Premium licensed.
+- Devices must be running [Fleetd](https://fleetdm.com/docs/using-fleet/orbit), the osquery manager from Fleet. Fleetd can be built with [fleetctl](https://fleetdm.com/docs/using-fleet/adding-hosts#osquery-installer).
+- Devices must be enrolled to an MDM solution.
```

Comment:
Out of curiosity, how many of the policies require an MDM solution?

Is this because we don't have the table that queries the actual/underlying setting, so we're relying on the MDM managed_policies table?

Reply:
> Out of curiosity, how many of the policies require an MDM solution?

Currently 29 queries require MDM.

> Is this because we don't have the table that queries the actual/underlying setting, so we're relying on the MDM managed_policies table?

Correct. And on some checks a configuration profile is the only (or best [*]) way to set a property.

[*] e.g. the PDF has comments like:
```
Note: AirDrop can only be enabled or disabled through configuration profiles. Any additional settings need to be set through the GUI or CL
```
or
```
Note: Since the profile method sets a system-wide setting and not a user-level one, the profile method is the preferred method. It is always better to set system-wide than per user.
```